app/funcs/csv_reader.py

```
@@ -80,11 +80,65 @@ def read_csv(self, file: Path, **kwargs) -> Generator[Dict[int, Dict], None, Non
+        Handles schema mismatches by reading row by row.
+        """
+        try:
+            # Use scan_parquet for lazy evaluation
+            lazy_df = pl.scan_parquet(file, **kwargs)
+            
+            # Collect the lazy frame and then iterate over rows
+            df = lazy_df.collect()
+            
+            # Use iter_rows to get each row as a dictionary
+            for row in df.iter_rows(named=True):
+                yield row
```

Ruff (0.14.7)

Replace `yield` over `for` loop with `yield from`

Replace with `yield from`

(UP028)